pkg/ddc/jindocache/transform.go

```diff
@@ -275,17 +275,22 @@ func (e *JindoCacheEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, m
 			cachesetPath = "local://" + ufsVolumesPath
 		}
 		cacheStrategy := "DISTRIBUTED"
-		metaPolicy := "ALWAYS"
+		metaPolicy := "ONCE"
 		readCacheReplica := 1
 		writeCacheReplica := 1
 
-		if mount.Options["cacheStrategy"] == "DHT" && mount.Options["metaPolicy"] == "ONCE" {
-			cacheStrategy = mount.Options["cacheStrategy"]
+		if mount.Options["metaPolicy"] == "ALWAYS" {
```

Comment:
For metaPolicy, it only checks for the "ALWAYS" case but not the "ONCE" case. You might want to add a more thorough check.
```golang
if userMetaPolicy, ok := mount.Options["metaPolicy"]; ok {
    if userMetaPolicy == "ONCE" || userMetaPolicy == "ALWAYS" {
         metaPolicy = userMetaPolicy
    } else {
         err = fmt.Errorf("Invalid metaPolicy: %s", userMetaPolicy)
         e.Log.Error(err, "Invalid metaPolicy")
         return err
    }
}
```